src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -84,7 +84,17 @@ class FileOutputWriter(props: Map[String, Object], outputFile: Option[File]) ext
             }
             catalog.refreshTable(tableName)
           }
-          case false => writer.saveAsTable(tableName)
+          case false => {
+            writer.save()
+            catalog.createTable(tableName, filePath)
+            fileOutputProperties.partitionBy match {
+              case Some(_) =>
+                log.info("Recovering partitions")
+                catalog.recoverPartitions(tableName)
+              case _ => None
```

Comment:
No need to None here

Reply:
removed none in both partition cases

Reply:
You can't remove the defualt case just don't return anything